### ISSUE TITLE
bmd: Font size should be set by styles, not by modifying document root font size

### ISSUE THREAD

**Issue by jonahkagan** [votingworks/vxsuite#1734]
## Summary
Currently, all of the styles in BMD use `rem` units, which are proportional to the root HTML document font size. Screens that we want to render with larger font have a `useEffectToggleLargeDisplay` trigger to modify the root document font size on render. Instead, they should use CSS styling to use a larger font size. This will require refactoring the existing styles to use `em` where appropriate so that they have font sizes relative to their parent instead of the document.

See: https://github.com/votingworks/vxsuite/pull/1733

**Comment by carolinemodic** [comment]
We should refactor this when we work on the accessibility features to implment more font sizes and themes.

**Comment by adghayes** [comment]
Note @kofi-q that your work may close this one as well.

**Comment by carolinemodic** [comment]
Closing this @kofi-q feel free to reopen if still relevant
